04-Addressing control failure/RemediationThroughScriptsUsingUI.md

```diff
@@ -0,0 +1,26 @@
+> The Azure Tenant Security Solution (AzTS) was created by the Core Services Engineering & Operations (CSEO) division at Microsoft, to help accelerate Microsoft IT's adoption of Azure. We have shared AzTS and its documentation with the community to provide guidance for rapidly scanning, deploying and operationalizing cloud resources, across the different stages of DevOps, while maintaining controls on security and governance.
+<br>AzTS is not an official Microsoft product – rather an attempt to share Microsoft CSEO's best practices with the community.
+<br>
+# AutoRemediation
+This feature is not enabled by default. If you have not enabled this feature in your AzTS setup yet, please follow steps mentioned below:
```

Comment:
Can we add screenshot how it will come in case of 'disabled' state as we are showing in last how it will show up in UI when flag is enabled.

Reply:
Added